extensions/cornerstone-dicom-sr/src/index.tsx

```diff
@@ -30,7 +31,37 @@ const dicomSRExtension = {
    * Only required property. Should be a unique value across all extensions.
    */
   id,
-  onModeEnter,
+
+  onModeEnter({ servicesManager }) {
+    const { toolbarService } = servicesManager.services;
+
+    toolbarService.addButtons([
+      {
+        // A base/default button for loading measurements. It is added to the toolbar below.
+        // Customizations to this button can be made in the mode or by another extension.
+        // For example, the button label can be changed and/or the command to clear
+        // the measurements can be dropped.
+        id: 'loadSRMeasurements',
+        component: props => (
+          <ViewportActionButton {...props}>{i18n.t('Common:LOAD')}</ViewportActionButton>
+        ),
+        props: {
+          commands: [
+            {
+              commandName: 'clearMeasurements',
+            },
+            {
+              commandName: 'loadSRMeasurements',
+            },
+          ],
+        },
```

Comment:
we can just put strings i guess now, no? commands: ['clearMeasurements', 'loadSRMeasurements']